Source/Core/Core/IOS/USB/Emulated/Skylander.cpp

```diff
@@ -19,465 +19,555 @@
+    {{451, 0x3000}, {"Thunderbolt (Trap Master)", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{451, 0x301D},
+     {"Thunderbolt (Clear) (Trap Master)", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{452, 0x3000}, {"Fling Kong", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{453, 0x3000}, {"Blades", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{453, 0x3403}, {"Blades (Legendary)", Game::TrapTeam, Element::Air, Type::Skylander}},
+    {{454, 0x3000}, {"Wallop (Trap Master)", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{455, 0x3000}, {"Head Rush (Trap Master)", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{455, 0x3402}, {"Head Rush (Nitro)", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{456, 0x3000}, {"Fist Bump", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{457, 0x3000}, {"Rocky Roll", Game::TrapTeam, Element::Earth, Type::Skylander}},
+    {{458, 0x3000}, {"Wildfire (Trap Master)", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{458, 0x3402},
+     {"Wildfire (Dark) (Trap Master)", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{459, 0x3000}, {"Kaboom (Trap Master)", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{460, 0x3000}, {"Trail Blazer", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{461, 0x3000}, {"Torch", Game::TrapTeam, Element::Fire, Type::Skylander}},
+    {{462, 0x3000}, {"Snap Shot (Trap Master)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{462, 0x450F},
+     {"Snap Shot (Virtual) (Trap Master)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{462, 0x3402},
+     {"Snap Shot (Dark) (Trap Master)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{463, 0x3000}, {"Lob Star (Trap Master)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{463, 0x3402},
+     {"Lob Star (Winterfest) (Trap Master)", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{464, 0x3000}, {"Flip Wreck", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{465, 0x3000}, {"Echo", Game::TrapTeam, Element::Water, Type::Skylander}},
+    {{466, 0x3000}, {"Blastermind (Trap Master)", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{467, 0x3000}, {"Enigma (Trap Master)", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{468, 0x3000}, {"Deja Vu", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{468, 0x3403}, {"Deja Vu (Legendary)", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{469, 0x3000}, {"Cobra Cadabra", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{469, 0x3402}, {"Cobra Cadabra (King)", Game::TrapTeam, Element::Magic, Type::Skylander}},
+    {{470, 0x3000}, {"Jawbreaker (Trap Master)", Game::TrapTeam, Element::Tech, Type::Skylander}},
+    {{470, 0x3403},
+     {"Jawbreaker (Legendary) (Trap Master)", Game::TrapTeam, Element::Tech, Type::Skylander}},
```

Comment:
Not sure why some of these have 2 sets of parentheses and some don't, but the names get incredibly long and the window only has a small space, so maybe either combine all of the double special words in to one, or maybe get rid of the (Giant), (Trap Master) etc as it makes the displayed name nearly impossible to read

Reply:
Going to add new entries to the type enum for those. Actually, probably for all gimmicks